src/mixins/audio_token_controls.js

```diff
@@ -15,6 +15,22 @@
   var playIconOffsetX = 50;
   var playIconOffsetY = 50;
 
+  var focusStrokeOuter = '#9c0d63';
+  var focusStrokeInner = '#ffffff';
```

Comment:
Can you make these properties of the Audio_token class? (That makes it easy to change from within amp-worksheet if we decide on a different color later...)